test/parse_tests/test_parse_datetime_uk.py

```
@@ -0,0 +1,538 @@
+        #             "2017-06-22 00:00:00", "мій день рождения був")
+        # testExtract("зустрінемось сьогодні вночі",
+        #             "2017-06-27 22:00:00", "зустрінемось вночі")
+        # testExtract("зустрінемось пізніше вночі",
+        #             "2017-06-27 22:00:00", "зустрінемось пізніше вночі")
+        # testExtract("яка будет погода завтра вночі",
+        #             "2017-06-28 22:00:00", "яка будет погода вночі")
+        # testExtract("яка будет погода наступного вівторок вночі",
+        #             "2017-07-04 22:00:00", "яка будет погода вночі")
+
+    def test_extract_ambiguous_time_uk(self):
+        morning = datetime(2017, 6, 27, 8, 1, 2, tzinfo=default_timezone())
```

Ruff

Local variable `morning` is assigned to but never used

Remove assignment to unused variable `morning`

(F841)